var/spack/repos/builtin/packages/samurai/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Samurai(CMakePackage):
+    """Intervals coupled with algebra of set to handle adaptive
+    mesh refinement and operators on it"""
+
+    homepage = "https://github.com/hpc-maths/samurai"
+    url = "https://github.com/hpc-maths/samurai.git"
```

Comment:
```suggestion
    homepage = "https://github.com/hpc-maths/samurai"
    git = "https://github.com/hpc-maths/samurai.git"
```

We should set the `git` property when pulling a package from a git repository instead of setting a url. 
[See the docs here for more detail.](https://spack.readthedocs.io/en/latest/packaging_guide.html#fetching-from-code-repositories)